src/components/ChatRoom.tsx

```diff
@@ -0,0 +1,80 @@
+import { Show, For, createSignal, useContext, onMount } from "solid-js";
+import log from "loglevel";
+import { AppContextContext } from "../AppContextProvider";
+import { Group, Message, MessageParams } from "../types";
+import * as api from '../api'
+import "./ChatRoom.css"
+
+type ChatRoomProps = {
+  group: Group
+};
+
+
+export default function ChatRoom(props: ChatRoomProps) {
+  let divRef: HTMLDivElement | undefined
+  const [formValue, setFormValue] = createSignal('')
+  const [appState] = useContext(AppContextContext);
+  const [openChatRoom, setOpenChatRoom] = createSignal(true);
```

Comment:
Maybe the chat should start out closed (false)

Reply:
nit:
I would also suggest "open" as opposed to "openChatRoom" just because it would be simpler in this context.
(openChatRoom sounds like a verb)

Reply:
renamed!